src/test/java/org/minions/devfund/royrodriguez/WhackAMoleTest.java

```diff
@@ -0,0 +1,231 @@
+package org.minions.devfund.royrodriguez;
+
+import org.junit.Test;
+
+import static junit.framework.TestCase.assertEquals;
+import static junit.framework.TestCase.assertTrue;
+import static org.junit.Assert.assertFalse;
+
+/**
+ * Tests for whack a mole game.
+ */
+public class WhackAMoleTest {
+
+    /**
+     * Tests constructor for whack a mole game.
+     */
+    @Test
+    public void whackAMoleConstructorTest() {
+        final int numAttempts = 5;
+        final int gridDimension = 10;
+        WhackAMole whackAMole = new WhackAMole(numAttempts, gridDimension);
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this object instantiation and use "WhackAMole.class" instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2133)